electron/services.js

```diff
@@ -109,25 +111,64 @@ export class Services {
         return Promise.all(shuttingDown);
     }
 
+    tryExistingRpc() {
+        return new Promise((resolve, reject) => {
+            check("http://localhost:8545").then((status) => {
+                if (!status.exists) {
+                    reject(new Error("Not exists"));
+                    return;
+                }
+                if (!this.setup.chain) {
+                    this.setup.chain = status.chain;
+                    resolve('setup from existing')
+                } else if (status.chain === this.setup.chain) {
+                    resolve('connect to existing')
+                } else {
+                    reject(new Error(`Wrong chain ${this.setup.chain} != ${status.chain}`))
```

Comment:
if geth is already available on 8545, shouldn't `setup.chain` be updated to match `status.chain`, instead of trying to start another instance of `geth`?